src/main/scala/cromwell/Main.scala

```diff
@@ -48,15 +50,24 @@ object Main extends App {
 
   def run(args: Array[String]): Unit = {
     if (args.length != 2) usageAndExit()
-    val runner = new SingleWorkflowRunner
-    val outputs = runner.run(new File(args(0)), new File(args(1)))
-    outputs match {
-      case Success(o) =>
-        import cromwell.binding.values.WdlValueJsonFormatter._
-        println("Workflow Completed.  Outputs are:")
-        println(o.toJson.prettyPrint)
-        runner.workflowManagerActor ! WorkflowManagerActor.Shutdown
-      case Failure(f) => println(f.printStackTrace())
+
+    try {
+      val wdl = FileUtil.slurp(Paths.get(args(0)))
+      val jsValue = FileUtil.slurp(Paths.get(args(1))).parseJson
+
+      val inputs: binding.WorkflowRawInputs = jsValue match {
+        case JsObject(rawInputs) => rawInputs
+        case _ => throw new RuntimeException("Expecting a JSON object")
+      }
+
+      val workflowManagerSystem = new WorkflowManagerSystem {}
+      val singleWorkflowRunner = SingleWorkflowRunnerActor.props(wdl, inputs, workflowManagerSystem.workflowManagerActor)
+      val actor = workflowManagerSystem.actorSystem.actorOf(singleWorkflowRunner)
+      // And now we just wait for the magic to happen
```

Comment:
So I guess the logic has changed so that SingleWorkflowRunner now prints stuff to the console instead of returning the outputs to the caller?  I think it was more flexible the other way... maybe instead of printing stuff out via the actor, we could have a way (via an Ask?) to get the outputs?  This would be nice because then SingleWorkflowRunner can  also be used as a Scala API that people could use instead of just something for the command line.